libsolidity/ast/Types.h

```diff
@@ -216,6 +216,9 @@ class Type
 	/// @returns the size of this data type in bytes when stored in memory. For memory-reference
 	/// types, this is the size of the memory pointer.
 	virtual unsigned memoryHeadSize() const { return calldataEncodedSize(); }
+	/// @returns the size of this data type in bytes when stored in memory. For memory-reference
+	/// types, this is the size of the actual data area, if it is statically-sized.
```

Comment:
Would "sum of element's head part's length" be a more accurate description / function name?